src/components/organisms/AssetActions/Compute/PriceOutput.tsx

```diff
@@ -62,35 +64,45 @@ export default function PriceOutput({
   hasPreviousOrderSelectedComputeAsset,
   hasDatatokenSelectedComputeAsset,
   algorithmPrice,
-  selectedComputeAssetTimeout
+  selectedComputeAssetTimeout,
+  loading
```

Comment:
nitpick(blocking): We should follow the convention for booleans here and use `isLoading` instead.